samples/core/src/main/resources/db/migration/h2/V0001__R001_Create_schema.sql

```diff
@@ -36,13 +36,13 @@ CREATE TABLE PRODUCT_AUD(
     pictureId BIGINT,
     alcoholic BOOLEAN,
     dtype VARCHAR(31) NOT NULL,
-    id BIGINT NOT NULL,
+    id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
*_AUD tables are for envers (to save historic revisions). In *_AUD tables there can never be added a new id so there should also not be an AUTO_INCREMENT here.